docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+services:
+  main:
+    command: bash
```

Comment:
It would make more sense if we launched the wott-agent in foreground mode (i know we don't have this feature yet).

Reply:
@vpetersson We do have this feature, but this container is for running agent manually for development, not for keeping it running all the time.

Reply:
Alright - fair enough.